less/_components/classes-grid.less

```diff
@@ -118,4 +118,15 @@
     left: 50%;
     transform: translate(-50%);
     min-height: 100px;
+}
+
+@media (min-width: 767px) {
+  .justify-left-desktop {
+    justify-content: left;
+  }
+}
+@media (max-width: 768px) {
```

Comment:
We should be using variables for this. Please see https://getbootstrap.com/docs/3.4/customize/#media-queries-breakpoints